app.js

```diff
@@ -0,0 +1,113 @@
+/**
+ * Created by dusting on 10/26/15.
+ */
+var getFile = require('./getFile');
+
+var investigation = getFile.processArgs(process);
+
+var graphDict = {};
+var graphDictArray = [];
+var touchedDict = {seen: [], starts:[], ends:[]};
+var pathDict = {paths:[]};
+
+witnessAccounts(investigation, findPaths);
+
+
+//This section maps nodes and edges to an object
+function witnessAccounts(investigation, callback) {
+    for (var people = 0; people < investigation.length; people++) {
```

Comment:
You are using an asynchronous pattern here, `callback` but i don't believe this function is doing anything that requires that. You should probably just return `witnessAccounts` then refactor this to be something like `getWitnessAccounts` so that line 13 can be.

```
var witnessAccounts = getWitnessAccounts(investigation);
```